mtmount/Dockerfile

```diff
@@ -0,0 +1,111 @@
+# This docker file cannot fully automate the EUI installation. You must carry on
+# with the installation of the VIPM package installs to complete the installation. 
+
+FROM centos:centos7
+
+LABEL maintainer Andrew Heyer <aheyer@lsst.org>
+
+ENV USER ${USER:-saluser}
+ENV HOME /home/saluser
+USER root
+# Create user and group
+RUN groupadd lsst && adduser -u 1004 -m -g lsst -s /bin/bash saluser
+
+COPY lsst-ts-nexus.repo /etc/yum.repos.d/lsst-ts-nexus.repo
+COPY wandisco-git.repo /etc/yum.repos.d/wandisco-git.repo
+RUN rpm --import http://opensource.wandisco.com/RPM-GPG-KEY-WANdisco
+RUN yum install -y git wget
+RUN yum install -y OpenSpliceDDS-6.9.0
+
+#Install and update various packages needed for Labview, VPM, and be able to
+#compile C++ tma management 
+#RUN yum update
+RUN yum groupinstall -y "Development Tools"
+RUN yum install -y cmake boost-devel.x86_64
+RUN wget http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+RUN rpm -ivh epel-release-latest-7.noarch.rpm
+RUN yum install -y geany java-1.7.0-openjdk-devel
+RUN yum install -y glibc.i686 libstdc++.so.6 libXinerama.i686 && \
+yum upgrade -y gnome-packagekit-common && \
+yum install -y libglvnd-glx-1.0.1-0.8.git5baa1e5.el7.i686
+RUN yum install -y yum-utils device-mapper-persistent-data lvm2
```

Comment:
it's much better and faster to group yum install together on a single line. Removes few sec of margin for starting & shutting yum. Also makes less commits, less docker files - and you really don't need the granulity

Reply:
Good points, changed to a single `RUN`